resemble.js

```diff
@@ -478,6 +478,16 @@ var _this = {};
 					if(hasMethod) { param(); }
 					return self;
 				},
+				setTolerance: function(value){
+
+					tolerance.red = value;
+					tolerance.green = value;
+					tolerance.blue = value;
+					tolerance.alpha = value;
+
```

Comment:
The reason I asked is because all the other functions set all the values, so it doesn't look like they are meant to be combined with one another. Also, I saw that your code is only invoking `setTolerance`, so I wondered if these other values will be uninitialized, or have values that just happen to work, but might change because they are not defined to be specific values.